gluten-ut/src/test/scala/org/apache/spark/sql/execution/joins/GlutenInnerJoinSuite.scala

```diff
@@ -17,8 +17,179 @@
 
 package org.apache.spark.sql.execution.joins
 
-import org.apache.spark.sql.GlutenSQLTestsBaseTrait
+import org.apache.spark.sql.{DataFrame, GlutenSQLTestsBaseTrait, Row}
+import org.apache.spark.sql.catalyst.expressions.Expression
+import org.apache.spark.sql.catalyst.optimizer.{BuildLeft, BuildRight, BuildSide}
+import org.apache.spark.sql.catalyst.planning.ExtractEquiJoinKeys
+import org.apache.spark.sql.catalyst.plans.logical.{Join, JoinHint}
+import org.apache.spark.sql.catalyst.plans.Inner
+import org.apache.spark.sql.execution.{joins, FilterExec, SparkPlan}
+import org.apache.spark.sql.execution.exchange.EnsureRequirements
+import org.apache.spark.sql.internal.SQLConf
 
 class GlutenInnerJoinSuite extends InnerJoinSuite with GlutenSQLTestsBaseTrait {
+  private val EnsureRequirements = new EnsureRequirements()
 
+  override def testNameBlackList: Seq[String] = Seq(
```

Comment:
Please move these into VeloxTestSettings because Velox and CH backend share this blacklist.

Reply:
Done.